src/Monotonic.cpp

```diff
@@ -443,7 +443,12 @@ class DerivativeBounds : public IRVisitor {
             } else {
                 delta.accept(this);
                 ConstantInterval rdelta = result;
-                adjusted_delta = multiply(rcond, rdelta);
+                // If rdelta is constant, we can't trust it, because if it were truly
```

Comment:
Given the hack/temporary nature of the fix, we should open a tracking issue and add a TODO comment with link to it here

Reply:
> I don't actually know it is a temporary hack. It might actually be correct.

Well, ok, but "This seems like a hack" and "Lgtm as a temporary fix" were in the comments just above...